pkg/globalnet/controllers/cluster_egressip_controller.go

```diff
@@ -254,6 +259,8 @@ func (c *clusterGlobalEgressIPController) allocateGlobalIPs(key string, numberOf
 		return true
 	}
 
+	metrics.RecordAllocateClusterGlobalEgressIPs(c.pool.GetCIDR(), numberOfIPs)
```

Comment:
I think we should move this after L279 then we don't need to record deallocated on L276 (it's a noop).